src/utils/sformat.fs

```diff
@@ -54,65 +54,73 @@ namespace Microsoft.FSharp.Text.StructuredFormat
 
     [<NoEquality; NoComparison>]
 #if COMPILER
+    type internal BoxRange = BoxRange of obj
```

Comment:
I'm afraid I can't do this one without help. I admit what I did here with the `#if` directives and attributes is a copy paste job without fully understanding what's going on.